src/_data/contactless/toc.yml

```diff
@@ -0,0 +1,66 @@
+toc:
+  - title: Prepare
+    subsection_items:
+      - title: Getting started
+        url: "getting-started"
+        paragraphs:
+          - title: How do I get started?
+          - text: Cal-ITP will provide the resource materials and technical support you need to navigate modernizing your fleet to accept contactless payments. To help us better assist you, we ask that you complete an <a href="https://www.camobilitymarketplace.org/contactless-support">interest form</a> to request Cal-ITP support. With this information, we can determine how best to support you with developing a project scope, getting buy-in from internal stakeholders, engaging with vendors, and implementing your new fare payments system.
+          - text: After you complete the form, we welcome you to browse the Contactless Payments MSA Information Packet (this guide) to understand the overall process and your next steps.
```

Comment:
What should `this guide` link to? @mrose914 

Is it https://docs.google.com/presentation/d/1XqLZWdsTlxG4W3j5kHu_QwC5QDDB3_c2/edit#slide=id.p1? And if so, do you want the link to a Google Slides URL, or to a PDF-version of this Google Slides, uploaded to cacamobilitymarketplace.org/?

Reply:
Updated https://resources.calitp.org/mobility-marketplace/Contactless-Payments-MSA-Information-Packet.pdf